src/components/base/JAssetSymbol/symbolsAvailable.js

```diff
@@ -0,0 +1,102 @@
+export const SYMBOLS_AVAILABLE = [
```

Comment:
would be better to place it in `src/data`

Reply:
It is not global data actually, but view parameters required only for symbol rendering. So they are context to their domain now.

It would be even better not to store it at all, but to get it automatically from file system.

But what do we say to such tasks? Not today!